docs/source/queries/lwt.md

```diff
@@ -1,32 +0,0 @@
-# Lightweight transaction (LWT) query
-
-A lightweight transaction query can be expressed just like any other query, via `Session`, with the notable difference of having an additional consistency level parameter - the `serial_consistency_level`.
```

Comment:
:warning: :question: It's not clear to my why this file was removed and whether its contents were pasted into some other file.